docker/Dockerfile.auto

```diff
@@ -0,0 +1,23 @@
+FROM pytorch/pytorch:2.1.0-cuda11.8-cudnn8-runtime
+
+# Set the working directory in the container
+WORKDIR /usr/src/app
+
+# Install build tools and compilers
+# Avoid user interaction with tzdata when installing
+ENV DEBIAN_FRONTEND=noninteractive
+
+# apply the relevant .env
+COPY ./env-samples/.env.auto.example ./.env
+
+# Copy the current directory contents into the container at /usr/src/app
+COPY . .
```

Comment:
DO NOT COPY EVERYTHING.
If i was not clear before. we have a lot of things that are unnecessary like docs, mds, images etc etc
copy only the files and folders required for running and installation + LISENCE

Reply:
fixed